sdk/nodejs/appservice/zMixins_http.ts

```diff
@@ -104,11 +135,18 @@ export class HttpEventSubscription extends mod.EventSubscription<mod.Context<Htt
             ...args,
             location,
             resourceGroupName,
+            hostSettings: {
+                ...args.hostSettings,
+                extensions: { http: args.extensionSettings },
+            },
         }, opts);
 
-        this.url = args.route
-            ? pulumi.interpolate`https://${this.functionApp.defaultHostname}/api/{${args.route}}`
-            : pulumi.interpolate`https://${this.functionApp.defaultHostname}/api/${name}`;
+        const routePrefix = args.extensionSettings && args.extensionSettings.routePrefix;
+        const rootPath = routePrefix === '' ? '' : `${routePrefix || 'api'}/`;

```

Comment:
can you use "double quotes"?  also, it woudlbe preferred to write this as `routePrefix === undefined ? "api" : routePrefix`.